cmd/local-gadget/containers/containers.go

```diff
@@ -118,19 +129,27 @@ func printContainers(parser *commonutils.GadgetParser[containercollection.Contai
 	return nil
 }
 
-func printContainer(parser *commonutils.GadgetParser[containercollection.Container], commonFlags utils.CommonFlags, container *containercollection.Container) error {
+func printEvent(parser *commonutils.GadgetParser[event], commonFlags utils.CommonFlags, event event) error {
 	switch commonFlags.OutputMode {
 	case commonutils.OutputModeJSON:
-		b, err := json.MarshalIndent(container, "", "  ")
+		b, err := json.MarshalIndent(event, "", "  ")
 		if err != nil {
 			return commonutils.WrapInErrMarshalOutput(err)
 		}
 		fmt.Printf("%s\n", b)
 	case commonutils.OutputModeColumns:
 		fallthrough
 	case commonutils.OutputModeCustomColumns:
-		fmt.Println(parser.TransformIntoColumns(container))
+		fmt.Println(parser.TransformIntoColumns(&event))
 	}
 
 	return nil
 }
+
+func eventFromPubSubEvent(e containercollection.PubSubEvent) event {
+	return event{
+		Timestamp: time.Now().Format(time.RFC3339),
+		Event:     e.Type.String(),
+		Container: *e.Container,
+	}
+}
```

Comment:
Yep, it should be like that. #1179 is looking ready so let me base my PR on that. Let's handle this in the same PR since I also wanted to use `containercollection.PubSubEvent` directly. Also, I am afraid it is easy to lose a change in which we just have to play with '&'/'*' :laughing: